NewSpring/ApollosProject
# Added Correct Share URLs

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR gives us share URLs for content items in the form of `https://newspring.cc/channelUrl/slug`.

![Screenshot 2019-06-17 13 43 46](https://user-images.githubusercontent.com/3229463/59625038-4ed62880-9106-11e9-9d9e-5586da998ebf.png)


### How do I test this PR?
Open up the sim and go to a content item.
Share said content item. 
Make sure that the URL that is shared is the correct URL that someone would use to find that item on newspring.cc.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
